README.md

```diff
@@ -1,13 +1,16 @@
 # Project Name
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The project objective was to work in pairs and build a chatbot.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The goal was to practice our conditional as well as function skills using vanilla javascript. We stuck to basic use of variables, functions and if-statements.
+
+We encountered several bugs relating to innerHTML that took us some time to wrap our heads around but in the end we managed to figure out a way to solve them.
+
+If we had more time we would for example have done some styling. We would also try other forms of inputs, such as radio buttons, images etc, regular buttons, etc.
 
 ## View it live
 
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Project can be found here:
+https://tender-dijkstra-73ca91.netlify.app/
```

Comment:
README file is a very special place in your repo. You should try to 'sell' your project here. 
Use it to tell your users what your application does, how to use it and eventually what next functionalities can be added in the future. 
Make it shine because this is cover of your project and you need to make your visitors stop for a while and look deeper to your code.